java/dictionary-renderer-graphics-overlay/README.md

```diff
@@ -0,0 +1,38 @@
+# Dictionary Renderer with Graphics Overlay
+
+Display mil2525d symbols.
```

Comment:
Lets just make it:

~Display mil2525d symbols.~
~ The dictionary renderer~ Creates ~these~ graphics using a local mil2525d style file and a XML file with key, value attributes for each graphic.